### ISSUE TITLE
insightio chrome plugin No Vertical ScrollBar

### ISSUE THREAD

**Issue by FreeLoopC** [lambdalab/UserFeedbacks#6]
insightio chrome plugin No Vertical ScrollBar and Horizen ScrollBar
It's not convenient to use it
For example

![1](https://user-images.githubusercontent.com/20551753/33640640-9b9e6396-da6c-11e7-9f1a-b0b5d96d9180.png)

**Comment by WangQianliang** [comment]
Version 0.7.8 doesn't seem to have this issue, maybe you should update your plugin.

**Comment by FreeLoopC** [comment]
Insight.io for Github 0.7.8
Improve GitHub code browsing experience by applying code intelligence from Insight.io.

Windows7 64bit

Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36

**Comment by FreeLoopC** [comment]
@WangQianliang

**Comment by mw-ding** [comment]
@FreeLoopC We reproduced the issue on Chrome 50.0 as you have reported. I would suggest you to upgrade Chrome, since 50.0 is fairly old. The up-to-date version is 62.0. Let us know if upgrading Chrome does not fix the issue.

**Comment by FreeLoopC** [comment]
I use insight web instead of insight chrome plugin now,  Insight web is good
but i still hope you fix this issue. Some projects don't have insight web mode

**Comment by mw-ding** [comment]
@FreeLoopC Thanks for coming back again. I am trying to understand the context. What do you mean by "insight web"? Are you talking about insight.io website instead of the Insight.io for GitHub Chrome Plugin?

**Comment by FreeLoopC** [comment]
web是指网站版

**Comment by FreeLoopC** [comment]
you are right

**Comment by mw-ding** [comment]
@FreeLoopC hey, do you still have the issue now?

**Comment by FreeLoopC** [comment]
The problem still exists,  Plugin Version 0.7.20
I use Octotree instead of it now
perhaps you can learn from Octotree to fix this issue

**Comment by FreeLoopC** [comment]
@mw-ding

**Comment by mw-ding** [comment]
@FreeLoopC Fair point. In the meantime, have you tried to upgrade to the latest version of Chrome?

**Comment by FreeLoopC** [comment]
@mw-ding Actually, I use 360 Jisu Chrome 8.5.0 version, chrome kernel version number 50.

As you know, the more new versions of domestic software are more hooligan,

so, I choose the old version to enjoy advertising-free.

I tried native chrome, but I'm not used to it.

In some way, native chrome is not more convenient than domestic chrome
